### ISSUE TITLE
Sharing images

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#167]
We need a page on sharing images. Create this after #166.

We will also need a stub page for openstack api interface and links to references, because right now the only way to share an image is to use the command line api.

**Comment by wwarriner** [comment]
We have seen maybe one request for this so far and it was handled ad hoc. The new plan here will be to make a note on an appropriate page that this is possible, and to contact support if it is needed. If the number of requests increases, or the next time we fulfill a request, we can write docs at that time.

**Comment by wwarriner** [comment]
Closed in favor of #325.
